src/groups/mqb/mqbblp/mqbblp_clusterproxy.cpp

```diff
@@ -784,7 +784,7 @@ void ClusterProxy::processEvent(const bmqp::Event&   event,
                        << "Received unexpected event: " << event;
         BSLS_ASSERT_SAFE(false && "Unexpected event received");
         return;  // RETURN
-    }  // break;
+    }            // break;
```

Comment:
These are strange changes---again, clang-format ones?  (and do we need them to pass CI?)